runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -141,6 +141,9 @@ public void invoke() {
                     throw new IllegalStateException("Orchestrator can not be selected");
                 }
 
+            } catch(RuntimeException e) {
+                log.warn("WorkflowRequest: Additional handling of the exception wrapped herein." +
+                        " Running {} on attempt {},  cause:", this, x, e);
```

Comment:
Minor thing. Since you touched this piece of code, would be great if you rename `x` to `retryIteration`

Reply:
Right! Better code readability. Pushing the change.